src/js/protocols/webstm32.js

```diff
@@ -27,7 +27,7 @@ function readSerialAdapter(event) {
 
 function onTimeoutHandler() {
     GUI.connect_lock = false;
-    console.log("Looking for capabilities via MSP failed");
+    console.log("[STM32] Looking for capabilities via MSP failed");
```

Comment:
Here it does not use the LogHead?

Reply:
This is a helper function outside of the class / prototype.

Reply:
`STM32Protocol.LogHead` will work? I've not looked at the code deeply.

Reply:
Yes tested change and works.